gammapy/irf/psf/map.py

```diff
@@ -523,6 +534,129 @@ def peek(self, figsize=(12, 10)):
         self.exposure_map.reduce_over_axes().plot(ax=axes[2], add_cbar=True)
 
         axes[3].set_title("Containment radius at 1 TeV")
-        self.containment_radius_map(energy_true=2 * u.TeV).plot(
+        kwargs = {self.energy_name:2 * u.TeV}
+        self.containment_radius_map(**kwargs).plot(
             ax=axes[3], add_cbar=True
         )
+
+
+class PSFMapReco(PSFMap):
+    """Class containing the Map of PSFs in reconstructed energy and allowing to interact with it.
+
+    Parameters
+    ----------
+    psf_map : `~gammapy.maps.Map`
+        the input PSF Map. Should be a Map with 2 non spatial axes.
+        rad and energy axes should be given in this specific order.
+    exposure_map : `~gammapy.maps.Map`
+        Associated exposure map. Needs to have a consistent map geometry.
+    """
+
+    tag = "psf_map"
```

Comment:
isn't the fact that the tag is the same as the `PSFMap` an issue? I am having trouble opening `PSFMapReco` objects from index tables because of it. But maybe I am doing something wrong. Shouldn't it be `psf_map_reco`?

Reply:
Yes, if we keep the `PSFMapReco` it should get a separate tag.

Reply:
done